yt/visualization/fixed_resolution.py

```diff
@@ -708,21 +712,37 @@ def __getitem__(self, item):
             deposition,
         )
 
-        bounds = []
-        for b in self.bounds:
-            if hasattr(b, "in_units"):
-                b = float(b.in_units("code_length"))
-            bounds.append(b)
+        dd = self.data_source.dd
 
         ftype = item[0]
-        x_data = self.data_source.dd[ftype, self.x_field]
-        y_data = self.data_source.dd[ftype, self.y_field]
-        data = self.data_source.dd[item]
+        if self.axis is None:
```

Comment:
Is `code_length` the right thing to do here?  I know we did it above, before, but I need a check on that -- should it be that, or `unitary`?

Reply:
Not sure why it matters either way--could you clarify?

Reply:
Not sure why it matters either way--could you clarify?

Reply:
I think I convinced myself it's OK, since everything will be in *the same* bounds.  I'm sorry for the noise.

Reply:
No, no--happy to have the thought.